downloads/swan-lake-release-notes/swan-lake-2201.3.5/RELEASE_NOTE.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ballerina-left-nav-release-notes
+title: 2201.3.5 (Swan Lake) 
+permalink: /downloads/swan-lake-release-notes/2201-3-5/
+active: 2201-3-5
+redirect_from: 
+    - /downloads/swan-lake-release-notes/2201-3-5
+    - /downloads/swan-lake-release-notes/2201.3.5/
+    - /downloads/swan-lake-release-notes/2201-3-5-swan-lake/
+    - /downloads/swan-lake-release-notes/2201-3-5-swan-lake
+---
+
+## Overview of Ballerina Swan Lake 2201.3.5
+
+<em>Swan Lake 2201.3.5 is the fourth patch release of Ballerina 2201.3.0 (Swan Lake Update 3) and it includes a new set of bug fixes to the language and tooling.</em>
```

Comment:
```suggestion
<em>Swan Lake 2201.3.5 is the fifth patch release of Ballerina 2201.3.0 (Swan Lake Update 3) and it includes a new set of bug fixes to the language and standard library.</em>
```

Reply:
```suggestion
<em>Swan Lake 2201.3.5 is the fifth patch release of Ballerina 2201.3.0 (Swan Lake Update 3) and it includes a new set of bug fixes to the language, tooling, and standard library.</em>
```
Shall I add it like this as we have some fixes in the tooling as well?